courses/models.py

```
@@ -78,3 +87,15 @@ class Meta:
         return f"{self.student.username} enrolled in {self.course.title}"
+
+
+class Module(BaseModel):
+    title = models.CharField(max_length=255)
+    course = models.ForeignKey(Course, on_delete=models.CASCADE, related_name='modules')
+    order = models.PositiveIntegerField(default=0)
+
+    class Meta:
+        ordering = ['order']
+
+    def __str__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)